retrodep/doc.go

```diff
@@ -36,11 +36,16 @@
 // searched. This function allows for repositories which are
 // collections of independently-vendored projects.
 //
-// The DescribeProject function takes a RepoPath and returns a
-// Representation, indicating the upstream version of the project or
-// vendored project, e.g.
+// The NewWorkingTree function makes a temporary local copy of the
+// upstream repository.
 //
-//     ref, rerr := retrodep.DescribeProject(proj, src.Path)
+//    wt, err := retrodep.NewWorkingTree(&proj.RepoRoot)
+//
+// The DescribeProject function takes a RepoPath, a WorkingTree and
```

Comment:
What about
s/ and$/,/
?
Note the duplicated 'and'

Reply:
Thanks, fixed.